weekly-updates/weekly-update.2015-07-03.md

```diff
@@ -0,0 +1,51 @@
+### io.js Week of July 3rd
+Important patches for io.js 1.8 and 2.3 and upcoming events.
+
+### io.js 1.8 and 2.3 Releases
+
+This week we have three io.js releases: [v2.3.2](https://iojs.org/dist/v2.3.3/) and two following important security patches [v1.8.3](https://iojs.org/dist/v1.8.3/) and [v2.3.3](https://iojs.org/dist/v2.3.3/), complete changelog from previous releases can be found [on GitHub](https://github.com/nodejs/io.js/blob/master/CHANGELOG.md) with the [v1.x changelog here](https://github.com/nodejs/io.js/blob/v1.x/CHANGELOG.md).
+
+### Notable Changes
+
+#### 1.8.3
+
+**Maintenance release**
+
+* **v8**: Fixed an out-of-band write in utf8 decoder. **This is an important security update** as it can be used to cause a denial of service attack.
+* **openssl**: Upgrade to 1.0.2b and 1.0.2c, introduces DHE man-in-the-middle protection (Logjam) and fixes malformed ECParameters causing infinite loop (CVE-2015-1788). See the [security advisory](https://www.openssl.org/news/secadv_20150611.txt) for full details. (Shigeki Ohtsu) [#1950](https://github.com/nodejs/io.js/pull/1950) [#1958](https://github.com/nodejs/io.js/pull/1958)
+* **build**:
+  - Added support for compiling with Microsoft Visual C++ 2015
+  - Started building and distributing headers-only tarballs along with binaries
+
+#### 2.3.2
+
+* **build**:
+  * Added support for compiling with Microsoft Visual C++ 2015
```

Comment:
Uh, does this mean you'd like it to be using `-` or `*` for the nested list items?

Oh, and worth pointing out, nested lists are not available in Medium too. It'll just be reformatted to the parent's list item.